From 5926161dc0de496a0a3420d8e3b9a035cc154c79 Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 19:04:49 -0400
Subject: [fs] btrfs: rename add_device_membership to btrfs_kobj_add_device

Message-id: <1407179109-13714-403-git-send-email-sandeen@redhat.com>
Patchwork-id: 87275
O-Subject: [RHEL7.1 PATCH 402/422] btrfs: rename add_device_membership to btrfs_kobj_add_device
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: Anand Jain <anand.jain@oracle.com>

commit 9b4eaf43f4b0207b5d1ca8b8d22df88ea9e142fe
Author: Anand Jain <anand.jain@oracle.com>
Date:   Tue Jun 3 11:35:59 2014 +0800

    btrfs: rename add_device_membership to btrfs_kobj_add_device

    Signed-off-by: Anand Jain <anand.jain@oracle.com>
    Reviewed-by: David Sterba <dsterba@suse.cz>
    Signed-off-by: Chris Mason <clm@fb.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/sysfs.c b/fs/btrfs/sysfs.c
index df39458..1395efb 100644
--- a/fs/btrfs/sysfs.c
+++ b/fs/btrfs/sysfs.c
@@ -605,7 +605,7 @@ static void init_feature_attrs(void)
  }
 }
 
-static int add_device_membership(struct btrfs_fs_info *fs_info)
+static int btrfs_kobj_add_device(struct btrfs_fs_info *fs_info)
 {
  int error = 0;
  struct btrfs_fs_devices *fs_devices = fs_info->fs_devices;
@@ -666,7 +666,7 @@ int btrfs_sysfs_add_one(struct btrfs_fs_info *fs_info)
  if (error)
   goto failure;
 
- error = add_device_membership(fs_info);
+ error = btrfs_kobj_add_device(fs_info);
  if (error)
   goto failure;
 
-- 
1.7.1